### PR TITLE
ticdc: update caveats of cyclic replication

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -742,9 +742,11 @@ sync-ddl = true
 2. 开启环形同步的数据表名字需要符合正则表达式 `^[a-zA-Z0-9_]+$`。
 3. 在创建环形同步任务前，开启环形复制的数据表必须已创建完毕。
 4. 开启环形复制后，不能创建一个会被环形同步任务同步的表。
-5. 如果想在线 DDL，需要确保以下两点：
-    1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
-    2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+5. 在多集群同时写入时，为了避免业务出错，请避免执行 DDL，比如 `ADDCOLUMN`/`DROP COLUMN` 等。
```
Comment (overvenus) [inline ticdc/manage-ticdc.md:745]:
```suggestion
5. 在多集群同时写入时，为了避免业务出错，请避免执行 DDL，比如 `ADD COLUMN`/`DROP COLUMN` 等。 
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -742,9 +742,11 @@ sync-ddl = true
 2. 开启环形同步的数据表名字需要符合正则表达式 `^[a-zA-Z0-9_]+$`。
 3. 在创建环形同步任务前，开启环形复制的数据表必须已创建完毕。
 4. 开启环形复制后，不能创建一个会被环形同步任务同步的表。
-5. 如果想在线 DDL，需要确保以下两点：
-    1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
-    2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+5. 在多集群同时写入时，为了避免业务出错，请避免执行 DDL，比如 `ADD COLUMN`/`DROP COLUMN` 等。
```
Comment (jackysp) [inline ticdc/manage-ticdc.md:745]: It's better to provide a whitelist of DDL. e.g. `ADD INDEX`, `DROP INDEX`.

Comment (overvenus) [reply ticdc/manage-ticdc.md:745]: Could you add your inputs using the review suggestion?


\`\`\` suggestion

\`\`\`

Comment (amyangfei) [reply ticdc/manage-ticdc.md:745]: Some DDLs such as `ADD INDEX` could work here, but we don't have enough tests for this feature, IMHO we can leave the enforcing ban of DDL here

**Review by amyangfei — APPROVED** [review]

**Review by leoppro — APPROVED** [review]

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (TomShawn) [inline ticdc/manage-ticdc.md:745]:
```suggestion
5. 在多集群同时写入时，为了避免业务出错，请避免执行 DDL 语句，比如 `ADD COLUMN`/`DROP COLUMN` 等。 
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -742,9 +742,11 @@ sync-ddl = true
 2. 开启环形同步的数据表名字需要符合正则表达式 `^[a-zA-Z0-9_]+$`。
 3. 在创建环形同步任务前，开启环形复制的数据表必须已创建完毕。
 4. 开启环形复制后，不能创建一个会被环形同步任务同步的表。
-5. 如果想在线 DDL，需要确保以下两点：
-    1. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
-    2. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
+5. 在多集群同时写入时，为了避免业务出错，请避免执行 DDL，比如 `ADD COLUMN`/`DROP COLUMN` 等。 
+6. 如果想在线 DDL，需要确保以下两点：
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:746]:
```suggestion
6. 如果想在线执行 DDL 语句，需要确保满足以下条件：
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -742,9 +742,11 @@ sync-ddl = true
...
+    1. 业务兼容 DDL 前后的表结构。
+    2. 多个集群的 TiCDC 构成一个单向 DDL 同步链，不能成环，例如示例中只有 C 集群的 TiCDC 关闭了 `sync-ddl`。
+    3. DDL 必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:749]:
```suggestion
    + 业务兼容 DDL 语句执行前后的表结构。
    + 多个集群的 TiCDC 组件构成一个单向 DDL 同步链，不能成环。例如以上在 TiDB 集群 A，B 和 C 上创建环形同步任务的示例中，只有 C 集群的 TiCDC 组件关闭了 `sync-ddl`。
    + DDL 语句必须在单向 DDL 同步链的开始集群上执行，例如示例中的 A 集群。
```

**Pull request merged**